sample_annotator/clients/nmdc/gold_client_wrapper.py

```diff
@@ -168,42 +168,50 @@ def transform_emp500_nmdc(
                 # depth, when depth can be retreived from GOLD API
                 if biosample["depthInMeters"] is not None:
                     depth = nmdc.QuantityValue(
-                                has_raw_value=biosample["depthInMeters"],
-                                has_numeric_value=biosample["depthInMeters"],
-                                has_unit="meter",
-                            )
+                        has_raw_value=biosample["depthInMeters"],
+                        has_numeric_value=biosample["depthInMeters"],
+                        has_unit="meter",
+                    )
                 else:
                     depth = {}
 
                 if biosample["depthInMeters2"] is not None:
                     depth2 = nmdc.QuantityValue(
-                                has_raw_value=biosample["depthInMeters2"],
-                                has_numeric_value=biosample["depthInMeters2"],
-                                has_unit="meter",
-                            )
+                        has_raw_value=biosample["depthInMeters2"],
+                        has_numeric_value=biosample["depthInMeters2"],
+                        has_unit="meter",
+                    )
                 else:
                     depth2 = {}
 
+                # retrieve INSDC identifier information using both projects and biosamples
+                insdc_biosample_identifiers = [
+                    proj["ncbiBioSampleAccession"]
```

Comment:
I think you need to prefix these.

Always check the schema you are writing against:

```yaml
  INSDC biosample identifiers:
    is_a: sample identifiers
    mixins:
      - INSDC identifiers
    aliases:
      - EBI biosample identifiers
      - NCBI biosample identifiers
      - DDBJ biosample identifiers
    pattern: "^biosample:SAM[NED]([A-Z])?[0-9]+$"
    multivalued: true
    description: identifiers for corresponding sample in INSDC
    examples:
      - value: https://identifiers.org/biosample:SAMEA5989477
      - value: https://identifiers.org/biosample:SAMD00212331
        description: I13_N_5-10 sample from Soil fungal diversity along elevational gradients
    see_also:
      - https://github.com/bioregistry/bioregistry/issues/108
      - https://www.ebi.ac.uk/biosamples/
      - https://www.ncbi.nlm.nih.gov/biosample
      - https://www.ddbj.nig.ac.jp/biosample/index-e.html
```

Reply:
Thank you Chris. I'll pay more attention to the schema from now.

But the resulting JSON data is being validated against the NMDC JSON Schema schema, so I thought it should have picked up on this validation error?